panoptes_client/panoptes.py

```diff
@@ -453,8 +453,10 @@ def find(cls, _id):
             )
 
     @classmethod
-    def paginated_results(cls, response, etag):
-        return ResultPaginator(cls, response, etag)
+    def paginated_results(cls, response, etag, **params):
+        _page_size = params.pop('page_size', '')
+        _page = params.pop('page', '')
```

Comment:
With these I would just call `params.pop` and not assign the result to a variable.